src/pages/EditMonitor.vue

```diff
@@ -1291,11 +1338,35 @@ message HealthCheckResponse {
                     this.monitor.port = "53";
                 } else if (this.monitor.type === "radius") {
                     this.monitor.port = "1812";
+                } else if (this.monitor.type === "snmp") {
+                    this.monitor.port = "161";
                 } else {
                     this.monitor.port = undefined;
                 }
             }
 
+            if (this.monitor.type === "snmp") {
+                // snmp is not expected to be executed via the internet => we can choose a lower default timeout
+                this.monitor.timeout = 5;
+            } else {
+                this.monitor.timeout = 48;
+            }
+
+            // Set default SNMP version
+            if (!this.monitor.snmpVersion) {
+                this.monitor.snmpVersion = "1";
+            }
+
+            // Set default jsonPath
+            if (!this.monitor.jsonPath) {
+                this.monitor.jsonPath = "$";
+            }
```

Comment:
Lets avoid this user error

```suggestion
            if (!this.monitor.jsonPath && this.monitor.type === "snmp") {
                this.monitor.jsonPath = "$";
            } else {
                this.monitor.jsonPath = "$.value";
            }
```